package.json

```diff
@@ -40,7 +40,8 @@
     "watch:scss:skip-standalone": "node-sass  scss/docs --functions sass-functions.js --output-style compressed --output build/css/docs --watch",
     "watch": "yarn build && yarn watch:scss",
     "clean": "rm -rf build docs/static/css node_modules/ yarn-error.log",
-    "percy": "percy exec -- node snapshots.js"
+    "percy": "percy exec -- node snapshots.js",
+    "svgs-to-mixins": "node scripts/convert-svgs-to-icon-mixins.js"
```

Comment:
Nitpicking a bit, but maybe would be more clear to make the package.json script name refer to icons, as that's precisely our use case (and we don't want someone to try convert any SVG).

So maybe `icon-svgs-to-mixins` or something like that, or `convert-icons`?